src/models/City.tsx

```diff
@@ -0,0 +1,14 @@
+export default class City {
+
+    id: number;
+    name: string;
+    coord: { lat: number, lon: number};
```

Comment:
I'd suggest extracting this TS type to avoid a duplicated code on line `8`. i.e.:
```
type CoordType: { lat: number, lon: number}
...

coord: CoordType;
```